### DISCUSSION TITLE
403 Forbidden when crawling an external website with spatie crawler.

### DISCUSSION THREAD

**Discussion by Calbo67** [spatie/crawler#363] [Q&A]
I've setup a crawler which works fine using spatie crawler on loads of different websites but on one website it returns a 403 which appears to be cloudflare blocking it. If I crawl this website through browsershot it crawls fine and gets past the cloudflare page, meaning it certainly isn't IP related. What could be causing this I'm not sure if it's something to do with the spatie crawler not having JS enabled but i've added the flag as shown in the image below. I noticed in the documentation you can add a setBrowserShot but not sure if this is something I need and what arguments I pass into if I do.

![image](https://user-images.githubusercontent.com/29214275/115976664-215f2f00-a568-11eb-9f61-08fecfa272fb.png)

**Comment by lucaros97** [comment]
Anyone has any update on this?
I've the same problem
